test/unit/map_traits.cpp

```diff
@@ -0,0 +1,70 @@
+//==================================================================================================
+/*
+  KUMI - Compact Tuple Tools
+  Copyright : KUMI Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#include "kumi/utils/concepts.hpp"
+#include <concepts>
+#include <type_traits>
+#define TTS_MAIN
+#include <kumi/tuple.hpp>
+#include <tts/tts.hpp>
+
+template<typename T>
+struct nary_traits : std::integral_constant<std::size_t, sizeof(T)>
+{};
+
+TTS_CASE("Check map_traits behavior")
+{
+  using mapped = kumi::tuple< std::integral_constant<std::size_t, 4>, std::integral_constant<std::size_t, 1>
```

Comment:
you wan tot use sizeof() there too or if not WASM and other 32 bits platforms will not pass the test